lib/vagrant-bindfs/cap/redhat/bindfs_install.rb

```diff
@@ -6,14 +6,17 @@ module BindfsInstall
 
           def self.bindfs_install(machine)
             machine.communicate.tap do |comm|
-              if comm.test('grep "CentOS release 6" /etc/centos-release')
-                comm.sudo('yum -y install fuse fuse-devel')
-                comm.sudo('wget http://bindfs.org/downloads/bindfs-#{VagrantPlugins::Bindfs::SOURCE_VERSION}.tar.gz -O bindfs.tar.gz')
-                comm.sudo('tar --overwrite -zxvf bindfs.tar.gz')
-                comm.sudo('[ -d ./bindfs ] && cd bindfs && ./configure && make && make install')
-                comm.sudo('ln -s /usr/local/bin/bindfs /usr/bin')
+
+              require "vagrant-bindfs/version"
+
+              if comm.test("grep \"CentOS release 6\" /etc/centos-release")
+                comm.sudo("yum -y install fuse fuse-devel")
+                comm.sudo("wget http://bindfs.org/downloads/bindfs-#{VagrantPlugins::Bindfs::SOURCE_VERSION}.tar.gz -O bindfs.tar.gz")
+                comm.sudo("tar --overwrite -zxvf bindfs.tar.gz")
+                comm.sudo("[ -d ./bindfs-#{VagrantPlugins::Bindfs::SOURCE_VERSION} ] && cd bindfs-#{VagrantPlugins::Bindfs::SOURCE_VERSION} && ./configure && make && make install")
```

Comment:
Even though you were naming the TAR file bindfs.tar.gz it doesn't change the name of the directory that's archived inside it. In this case, the TAR file included the bindfs-[SOURCE_VERSION] directory.